tests/v2_validation/cattlevalidationtest/core/test_webhook.py

```diff
@@ -0,0 +1,116 @@
+from common_fixtures import *  # NOQA
+import requests
+from requests.auth import AuthBase
+import json
+
+access_key = os.environ.get("CATTLE_ACCESS_KEY", 'admin')
+secret_key = os.environ.get("CATTLE_SECRET_KEY", 'admin')
+
+def test_construct_webhook_test_max(client):
+    # Create Environment
+    env = client.create_stack(name=random_str())
+    env = client.wait_success(env)
+    assert env.state == "active"
+
+    # Create Service of scale 1
+    service_name = random_str()
+
+    image_uuid = TEST_IMAGE_UUID
+    launch_config = {"imageUuid": image_uuid}
+    service = client.create_service(name=service_name,
+    	                            stackId=env.id,
+                                    launchConfig=launch_config)
+    service = client.wait_success(service)
+    assert service.state == "inactive"
+    service.activate()
+    service = client.wait_success(service)
+    assert service.state == "active"
+
+    # Create webhook to scale up service by 2
+    url = "http://localhost:8080/v1-webhooks/receivers?projectId="+env.accountId
+    data = {
+        "name": "newservice 1 up-2",
+        "driver": "scaleService",
+        "scaleServiceConfig": {
+            "action": "up",
+            "amount": 2,
+            "serviceId": service.id,
+            "min": 1,
+            "max": 4
+        }
+    }
+    headers = {"Content-Type": "application/json", "Accept": "application/json"}
+    # r = requests.post(url, auth=(env.access_key, env.secret_key), data=json.dumps(data), headers=headers)
```

Comment:
@cjellick I left this for the implementation with auth turned on that I was asking you about. I was going to mention this in a comment on PR.